components/onboarding-modal/OnboardingProfileCard.js

```diff
@@ -2,35 +2,84 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Flex, Box } from '@rebass/grid';
 import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
 
 import Avatar from '../../components/Avatar';
 import StyledTag from '../../components/StyledTag';
+import StyledTooltip from '../../components/StyledTooltip';
 
 const Admin = styled(StyledTag)`
   font-size: 14px;
   border-top-right-radius: 50px;
   border-bottom-right-radius: 50px;
+  text-transform: none;
+  display: flex;
+  align-items: center;
 `;
 
 class OnboardingProfileCard extends React.Component {
   static propTypes = {
     collective: PropTypes.object,
+    adminCollective: PropTypes.object,
+    removeAdmin: PropTypes.func,
   };
 
   render() {
-    const { collective } = this.props;
-    const { name } = collective;
+    const { collective, removeAdmin, adminCollective } = this.props;
+    const { name, email } = collective;
 
     return (
       <Flex my={1} ml={2}>
-        <Admin textTransform="none">
-          <Flex alignItems="center">
-            <Avatar radius={15} collective={collective} />
-            <Box fontSize="Caption" ml={2}>
-              {name}
-            </Box>
-          </Flex>
-        </Admin>
+        {/* for invited users email is null */}
+        {email === null ? (
+          <StyledTooltip
+            content={() => <FormattedMessage id="onboarding.admins.pending" defaultMessage="Pending for approval" />}
+          >
+            <Admin
+              closeButtonProps={{
+                closeButtonWidth: '1.5em',
+                closeButtonHeight: '1.5em',
+                closeButtonColor: 'black',
+                closeButtonBackground: 'rgba(0, 0, 0, 0)',
+                closeButtonDisplay: 'flex',
+                closeButtonAlign: 'center',
+                onClick: () => {
+                  removeAdmin(collective);
+                },
+              }}
+            >
+              <Flex alignItems="center">
+                <Avatar radius={15} collective={collective} />
+                <Box fontSize="Caption" ml={2}>
+                  {name} {email === null}
```

Comment:
I don't understand this `{email === null}`?

Reply:
Sorry, that should not still be there. I had some issues with a rebase. Will remove.